src/components/ApplyFlow.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+
+export default function ApplyFlow(props) {
```

Comment:
unless there's some benefit to passing all props into a component (usually when cascading through a hierarchy of children), I typically prefer to see whatever explicit props that component requires:

`export default function ApplyFlow({ applyFlow, handleFlow }){`

Reply:
Got it. I've gone back and refactored this so that the props are destructured. Should I take a look at all the other components and refactor?

Reply:
no I think for the purpose of this PR, this is good enough for now